### PR TITLE
Registry Service clean-up

### PR DESCRIPTION
**Pull request by duglin**
Add some missing stuff to make it more real

Signed-off-by: Doug Davis <dug@microsoft.com>

### CODE REVIEW

File: registry/spec.md [hunk 1]
```diff
@@ -1,57 +1,289 @@
...
+  - `John Smith`
+  - `john.smith@example.com`
+
+### `modifiedOn`
+- Type: Timestamp
+- Description: The date/time of when the entity was last updated.
+- Constraints:
+  - MUST be an [RFC3339](https://tools.ietf.org/html/rfc3339) timestamp
+- Examples:
+  - `2030-12-19T06:00:00Z"
```
Comment (JemDay) [inline registry/spec.md:243]: nit: Inconsistent use of ' and "

File: registry/spec.md [hunk 2]
```diff
@@ -1,57 +1,289 @@
...
+  - `John Smith`
+  - `john.smith@example.com`
+
+### `createdOn`
+- Type: Timestamp
+- Description: The date/time of when the entity was created.
+- Constraints:
+  - MUST be an [RFC3339](https://tools.ietf.org/html/rfc3339) timestamp
+- Examples:
+  - `2030-12-19T06:00:00Z"
```
Comment (JemDay) [inline registry/spec.md:223]: nit: Inconsistent use of ' and "

**Review by JemDay — CHANGES_REQUESTED** [review]

File: registry/spec.md (hunk 2 above)
Comment (duglin) [reply registry/spec.md:223]: LOL took me a while to realize you wrote "and" not "an". I just couldn't find "and" in this section!
Thanks for noticing my copy-n-paste error.

File: registry/spec.md (hunk 1 above)
Comment (duglin) [reply registry/spec.md:243]: fixed

**Comment by duglin** [comment]
Approved on the 2/16 call

**Pull request merged**